sbt-plugin/src/main/scala/com/akkaserverless/sbt/AkkaserverlessPlugin.scala

```diff
@@ -57,10 +57,11 @@ object AkkaserverlessPlugin extends AutoPlugin {
   import autoImport._
 
   val AkkaServerlessSdkVersion = BuildInfo.version
+  val AkkaServerlessProtocolVersion = BuildInfo.protocolVersion
 
   override def projectSettings: Seq[sbt.Setting[_]] = Seq(
     libraryDependencies ++= Seq(
-      "com.akkaserverless" % "akkaserverless-sdk-protocol" % "0.7.1" % "protobuf-src",
```

Comment:
`AkkaServerless.FrameworkVersion` is already on 0.8.3, but we hardcoded the sdk to 0.7.1.